x11-misc/polybar/polybar-3.1.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="A fast and easy-to-use tool for creating status bars."
+HOMEPAGE="https://github.com/jaagr/polybar"
+EGIT_REPO_URI="https://github.com/jaagr/polybar.git"
+EGIT_CLONE_TYPE="shallow"
+EGIT_COMMIT="${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="x86 amd64"
+
+IUSE="alsa i3wm mpd github +network"
+RDEPEND="
+	x11-libs/libxcb
+	dev-lang/python:2.7
+	x11-proto/xcb-proto
```

Comment:
Hi, I made the same PR basically - but removed the previous version and fixed this one dependency; the category has to be `x11-base`. See https://github.com/das-labor/overlay/pull/30/files :)